src/Combinatorics/SimplicialComplexes.jl

```diff
@@ -0,0 +1,275 @@
+end
+
+function SimplicialComplex(generators::Vector{Set{Int}})
+    K = Polymake.topaz.SimplicialComplex(INPUT_FACES=generators)
+    SimplicialComplex(K)
+end
+
+################################################################################
+##  Auxiliary
+################################################################################
+
+function _vertexindices(K::Polymake.BigObject)
+    if Polymake.exists(K,"VERTEX_INDICES")
+        return Vector{Int}(K.VERTEX_INDICES)
+    else
+        return Vector{Int}(1:K.N_VERTICES)
+    end
+end
+
+_reindexset(M::Set{Int}, ind::Vector{Int}) = [ ind[x+1] for x in M ]
+
+function _characteristicvector(M::Vector{Int}, n::Int)
+    chi = zeros(Int, n)
+    chi[M] .= 1
+    return chi
+end
+
+################################################################################
+##  Properties
+################################################################################
+
+@doc Markdown.doc"""
+    nvertices(K::SimplicialComplex)
+
+Number of vertices of the abstract simplicial complex `K`.
+"""
+nvertices(K::SimplicialComplex) = pm_object(K).N_VERTICES
+
+@doc Markdown.doc"""
+    facets(K::SimplicialComplex)
+
+Maximal (by inclusion) faces of the abstract simplicial complex `K`.
+"""
+function facets(K::SimplicialComplex)
+    bigobject = pm_object(K)
+    the_facets = Vector{Set{Int}}(bigobject.FACETS)
+    return [ [x+1 for x in sigma] for sigma in the_facets ] # shift polymake indices by one
+end
+
+@doc Markdown.doc"""
+    dim(K::SimplicialComplex)
+
+Dimension of the abstract simplicial complex `K`.
+"""
+dim(K::SimplicialComplex) = pm_object(K).DIM
+
+@doc Markdown.doc"""
+    f_vector(K::SimplicialComplex)
+
+Face vector (number of faces per dimension) of the abstract simplicial complex `K`.
+"""
+f_vector(K::SimplicialComplex) = Vector{Int}(pm_object(K).F_VECTOR)
+
+@doc Markdown.doc"""
+    f_vector(K::SimplicialComplex)
+
+H-vector of the abstract simplicial complex `K`.
+"""
+h_vector(K::SimplicialComplex) = Vector{Int}(pm_object(K).H_VECTOR)
+
+@doc Markdown.doc"""
+    betti_numbers(K::SimplicialComplex)
+
+Rational Betti numbers of the abstract simplicial complex `K`.
+"""
+betti_numbers(K::SimplicialComplex) = Polymake.topaz.betti_numbers(pm_object(K))
+
+@doc Markdown.doc"""
+    euler_characteristic(K::SimplicialComplex)
+
+Reduced Euler characteristic of the abstract simplicial complex `K`.
+"""
+euler_characteristic(K::SimplicialComplex) = pm_object(K).EULER_CHARACTERISTIC
+
+@doc Markdown.doc"""
+    minimalnonfaces(K::SimplicialComplex)
+
+Minimal non-faces of the abstract simplicial complex `K`.
+
+# Example
+```jldoctest
+julia> K = SimplicialComplex([[1,2,3],[2,3,4]]);
+
+julia> minimalnonfaces(K)
+1-element Vector{Vector{Int64}}:
+ [1, 4]
+```
+"""
+function minimalnonfaces(K::SimplicialComplex)
+    bigobject = pm_object(K)
+    mnf = Vector{Set{Int}}(bigobject.MINIMAL_NON_FACES)
+    return [ [x+1 for x in f] for f in mnf ] # shift polymake indices by one
```

Comment:
Do you want the return type to be of `Vector{Set{Int}}` too or will `Vector{Vector{Int64}}` do as in the example? At any rate, the following would work in the first case:
```
julia> Vector{Set{Int}}(Polymake.to_one_based_indexing(K.pm_simplicialcomplex.MINIMAL_NON_FACES))
1-element Vector{Set{Int64}}:
 Set([4, 1])
```

Reply:
Here I am a bit uncertain.

While the objects are sets mathematically, I do not like Julia's `Set` data type too much.  Here it only creates an overhead. Most function I am using tend to return a `Vector`.  Going back and forth between `Vector` and `Set` looks like wasteful to me.  That could be a topic for a general discussion.